Sprintfinity3902/Dungeon/Room.cs

```diff
@@ -34,6 +36,8 @@ public Room(string fileLocation, int id)
             garbage = new List<IEntity>();
             enemyProj = new List<IEntity>();
             doors = new List<IDoor>();
+            RoomPos = new Point(2,7);
+            RoomType = 0;
```

Comment:
I know that first line is my code, but I am not sure assigning these to random values would be wise in the future. (Also mine are magic numbers).

Did this have to be assigned here? what If the game tries to create a room of type 0?